tests/posix/common/src/pthread.c

```diff
@@ -371,6 +378,63 @@ ZTEST(pthread, test_pthread_termination)
 	zassert_equal(ret, ESRCH, "cancelled a terminated thread!");
 }
 
+ZTEST(pthread, test_pthread_tryjoin)
+{
+	pthread_t alive_1_s = {0};
+	void *retval;
+
+	/* Creating a threads that exits after 1s*/
+	zassert_ok(pthread_create(&alive_1_s, NULL, thread_alive_1_s, NULL));
+
+	/* Attempting to join immediately should fail */
+	zassert_equal(pthread_tryjoin_np(alive_1_s, &retval), EBUSY);
+
+	/* Sleep so alive_1_s-thread will exit */
+	sleep(ONE_SECOND);
+	sleep(ONE_SECOND);
```

Comment:
This could work, I guess.
```suggestion
	usleep(200 * USEC_PER_MSEC);
```